draft-ietf-cats-framework.md

```diff
@@ -381,13 +380,63 @@ Note that, depending on the design considerations and service requirements, per-
+
+## Verify Correct Operations
+
+CATS may be implemented by extending some existing control plane protocols, such as BGP or PCEP. A CATS implementation must log error events for better network management and operation. Means to assess the reachability and trace CATS paths should be supported.
+
+## Impact on Network Operations
+
+Computing metrics are collected and distributed in CATS. A new function is needed to be deployed to manage the cooperation between network elements and computing elements. For example, this function may be provided by an orchestrator connecting with C-SMA and C-NMA. This might bring more complexity of the network management, especially if this function is not leveraged for other purposes beyond CATS.
+
 # CATS Framework Workflow {#sec-cats-workflow}
 
 The following subsections provide an overview of how the CATS workflow operates.
 
 ## Service Announcement
 
-A service is associated with a unique identifier called a CS-ID. A CS-ID may be a network identifier, such as an IP address. The mapping of CS-IDs to network identifiers may be learned through a name resolution service (e.g., DNS {{?RFC1034}}). Note that CATS framework does not assume nor preclude any specific name resolution service.
+In order to provide service to clients, a service needs to be known by the clients before the clients requesting for service. Typically, in the current practices of the web services, a service provider will register the IP addresses of the servers in a DNS server. A client can request for server IP addresses by the URL, and it can receive the IP addresses from the DNS server.
+
+In CATS framework, a service is associated with a unique identifier called a CS-ID by the service provider when deploying the service. Depending on the implementation, a CS-ID may be a network identifier, such as an IP address of servers, or other new defined IDs. If the CS-ID is not the server IP address, then the mapping of the CS-ID to network identifiers may be learned through a name resolution service (e.g., DNS {{?RFC1034}}), which will require for further protocol extensions and may be described in other documents. Note that CATS framework does not assume nor preclude any specific name resolution service. When a CS-ID is the server IP address, the service annoucement is not changed in the CATS framework comparing to the existing services. Therefore, this document suggest to use the IP addrees as the CS-ID in order to simplify the implementation.
```

Comment:
We need to clarify how/whether CS-ID is different from the service identifier used by client to access a service?